draft-ietf-core-coap-tcp-tls.md

```diff
@@ -178,13 +168,10 @@ hand, does not provide such information to a NAT and timeouts tend to be much
 shorter {{HomeGateway}}.
 
 Some environments may also benefit from the ability of TCP to exchange
-larger payloads such as firmware images without application layer
-segmentation and to utilize the more sophisticated congestion control
+larger payloads, such as firmware images, without application layer
+fragmentation and to utilize the more sophisticated congestion control
 capabilities provided by many TCP implementations.
 
-(Note that there is ongoing work to add more elaborate congestion control
-to CoAP as well, see {{-cocoa}}.)
-

```

Comment:
Actually, that draft is close to WG adoption (we have waited a while for the research to finish).  By the time we finish TCP/TLS, it should be a WG draft.